datapipelines/src/main/kotlin/io/customer/sdk/DataPipelineInstance.kt

```diff
@@ -253,5 +256,5 @@ abstract class DataPipelineInstance : CustomerIOInstance {
     /**
      * Delete the currently registered device token
      */
-    abstract fun deleteDeviceToken()
+    abstract fun deleteDeviceToken(enrichment: EnrichmentClosure? = null)
```

Comment:
Because this one is public, I think we should add `@JvmOverloads` here just to be safe